src/components/SharedComponents/PhotoCount.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { View } from "components/styledComponents";
+import * as React from "react";
+import Svg, { Path, Text } from "react-native-svg";
+
+type Props = {
+  count: number,
+  size?: number,
+  shadow: boolean,
+};
+
+const PhotoCount = ( { count, size, shadow }: Props ): React.Node => {
+  if ( count <= 0 || count > 99 ) {
+    return null;
+  }
+
+  return (
+    <View
+      className={`drop-shadow-md w-[${String( size )}px] h-[${String( size )}px] ${String(
+        shadow && "shadow"
+      )}`}
+    >
+      <Svg
+        height={size}
+        width={size}
+        viewBox="0 0 24 24"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <Path
+          fillRule="nonzero"
+          clipRule="evenodd"
+          d="M4 5.818a4 4 0 00-4 4V20a4 4 0 004 4h10.182a4 4 0 004-4V9.818a4 4 0 00-4-4z"
+          fill="#fff"
+          filter="url(#drop-shadow)"
+        />
+        <Path
+          // eslint-disable-next-line
+          d="M15.364 3.636h-9.53A4 4 0 019.818 0H20a4 4 0 014 4v10.182a4 4 0 01-3.636 3.984v-9.53a5 5 0 00-5-5z"
+          fill="#fff"
+          clipRule="evenodd"
+          fillRule="nonzero"
+          filter="url(#drop-shadow)"
+        />
+        <Text x="38%" y="77%" textAnchor="middle" fontSize="10" fill="gray">
```

Comment:
Is it possible to use our already styled component Body3 here?

Reply:
Yes it s possible to use Body3 but :
```
When you use the Text component from react-native inside an Svg component created with react-native-svg, the text is rendered as a bitmap image, which means that it doesn't have the same level of scalability and interactivity as other vector elements created with react-native-svg. If you need to use scalable text in your SVG graphics, it's better to use the Text component from react-native-svg
```

Reply:
I would like to avoid having to add any styles here to this Text, and instead, use our already existing styled Text component that fits the design specs here. Then we have a central place to change if needed.
Do I understand you correctly that as long as we do not change the size of the icon or count number during runtime we can use Body3 because the bitmap would not need to change because we would not scale or interact with it? Is there a performance impact in terms of having the bitmap?

Reply:
by using Text from the svg dep the font size for example it s managed by the size of the svg component.
I tried using react native Text component and I did't find a way to make the required stype.
Please feel free to try and commit in the branch

Reply:
Can you please position the new Body3 element correctly in the svg.

Reply:
That was possible only by adding a react-native component inside ForeignObject react-native-svg component